### PR TITLE
docs: regenerates with new docgen changes

### PR DESCRIPTION
**Pull request by Conni2461**
and 1 builtin example.

Someone can review and if its good either merge it or finish it :) But it has some changes in in the documentation that needs to be included and i already did this when developing the docgen changes so here it is.

### CODE REVIEW

**Comment by Conni2461** [comment]
This already happend. I will do the formatting fixes in the next PR

File: lua/telescope/builtin/init.lua [hunk 1]
```diff
@@ -25,6 +29,19 @@ local builtin = {}
...
+--- Later can be enabled with a different `find_command`.
+---@param opts table: Configure behavior of `find_files`.
+---@field find_command table: Specify your find command as a table of strings.
+---@field hidden boolean: Enable or disable hidden files. Default is false.
+---@field follow boolean: Enable or disable if symbolic links should be followed. Default is false.
+---@field cwd string: Specify the path on which find_files should run.
+---@field search_dirs table: Specify multiple search directories as table of paths.
+---@field entry_maker function: How entries are generated. Can be changed if you want a different display output.
+---@field disable_devicons boolen: Enable or disable devicons in front of each entry. Default is true if nvim-web-devicons is installed.
+---@field shorten_path boolean: Enable or disable if paths should be shortened. Default is false.
```
Comment (Conni2461) [inline lua/telescope/builtin/init.lua:44]: CC @smithbm2316 @fdschmidt93

That would be an example of how we could document builtins :) multiline `@param`/`@field` is will be done soon.

Comment (smithbm2316) [reply lua/telescope/builtin/init.lua:44]: Thanks for the example! Sorry I've been a bit absent on this lately, I'm finishing up my last couple of days of my undergrad degree and I'll hopefully be back able to work on docs starting next week!

Comment (fdschmidt93) [reply lua/telescope/builtin/init.lua:44]: Very nice! I'll also try to get started writing up docs on my end towards end of this week or beginning of next week :)

**Pull request closed without merging**